metadata.json

```diff
@@ -98,6 +98,6 @@
   "dependencies": [
     {"name":"ghoneycutt/common","version_requirement":">= 1.4.1 < 2.0.0"},
     {"name":"ghoneycutt/nsswitch","version_requirement":">= 1.3.0 < 2.0.0"},
-    {"name":"puppetlabs/stdlib","version_requirement":">= 4.6.0 < 5.0.0"}
+    {"name":"puppetlabs/stdlib","version_requirement":">= 4.13.1 < 5.0.0"}
```

Comment:
Curious is there something special about 4.13.1 that has functionality we are using?

Reply:
I think I pulled that from some official puppetlabs modules that got updated to only support Puppet 4/5.  The 4.13.0 is needed for custom Stdlib data types, like the one for validating absolute path.